lib/debug/session.rb

```diff
@@ -401,27 +401,27 @@ def process_command line
       case cmd
       ### Control flow
 
-      # * `s[tep]`
+      # * => `s[tep]`
```

Comment:
Even if you'd ultimately substitute it, require any existing and future command comment to have additional characters is not the best solution to help you colorize them. You should try other approaches like identifying an alternative pattern instead.

Reply:
All the changes made in the docs were reverted.

Thank you.